dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/TestCases/TestCaseFixedOmegaRef.mo

```diff
@@ -0,0 +1,97 @@
+within Dynawo.Examples.Nordic.TestCases;
```

Comment:
I think this test case can be deleted. Please keep it locally for the moment though.

Reply:
Kept on my machine